app/controllers/admin/employees_controller.rb

```diff
@@ -54,15 +55,16 @@ def destroy
 
     def create_params
       params.require(:employee)
-            .permit(:name, :email, :role, :password, :password_confirmation)
+            .permit(:name, :email, :role, :password, :password_confirmation, :lat, :long)
     end
 
     def edit_params
-      params.require(:employee).permit(:name, :email, :role, :profile_picture)
+      params.require(:employee).permit(:name, :email, :role, :profile_picture, :lat, :long)
     end
 
     def fetch_employee
       @employee = Employee.find(params[:id])
+      @admin = true if @employee.role == 'Admin'
```

Comment:
You can simplify this line by removing the `if` clause here. `@employee.role == 'Admin'` returns `true` so you can use that as the value for variable setting like so: 
```
> a = "yes"
> b = a == "yes"

> puts b
=> true
```